### DISCUSSION TITLE
Lesson 6: env WEB3_INFURA_PROJECT_ID value error

### DISCUSSION THREAD

**Discussion by Riyazmansuri** [smartcontractkit/full-blockchain-solidity-course-py#1438] [Q&A] [answered]
when adding the project id environment variable into the env file it's showing error
![enverror](https://user-images.githubusercontent.com/40120512/164885532-896adb9f-e265-41fe-b2e3-2229ad82a958.PNG)
when compiling it's showing 
![error](https://user-images.githubusercontent.com/40120512/164885601-2ba15335-4be6-4070-9554-34e98768e32c.PNG)

**Comment by cromewar** [comment, marked as answer]
As you are on windows, you should add the infura key on path trough windows advanced configuration.
